agentextension/build.gradle.kts

```diff
@@ -40,6 +40,8 @@ tasks {
       include("LICENSE")
       include("NOTICE")
       include("licenses/**")
+
+      // TODO : ignore classes that must remain in the agent as we do in 'relocateJavaagentLibs'
```

Comment:
Is this todo still required?

Reply:
Yes, I think so, if you look at the content of the built extension jar it contains the API, SDK and the semantic conventions, I plan to do this in a separate PR though.

Reply:
Ah, I see. Thanks for clarifying.